openapi/openapi.yaml

```diff
@@ -257,6 +305,98 @@ paths:
 
 components:
   schemas:
+
+    Collection:
+      title: 'Collection'
+      description: 'Ansible content collection'
+      type: object
+      properties:
+        name:
+          description: 'Collection name. Must be lower case containing
```

Comment:
You can use yaml syntax for multiline strings, like:

```
description: |
    foo
    bar
    baz
```

Reply:
updated